media_softlet/linux/common/os/i915_production/mos_bufmgr.c

```diff
@@ -60,6 +60,9 @@
 #include "libdrm_lists.h"
 #include "mos_bufmgr.h"
 #include "mos_bufmgr_priv.h"
+#ifdef ENABLE_NEW_KMD
```

Comment:
NEW_KMD sounds general. @XinfengZhang what's the flag you prefer to use, NEW_KMD or Xe_KMD or something else?

Reply:
I remmebered the macro named XE_KMD at the beginning, but it was changed to NEW_KMD, not sure of the reason...